pkg/work/spoke/spokeagent.go

```
@@ -35,18 +35,7 @@ import (
-	// If a controller queue size is too large (>500), the processing speed of the controller will drop significantly
-	// with one worker, increasing the work numbers can improve the processing speed.
-	// We compared the two situations where the worker is set to 1 and 10, when the worker is 10, the resource
-	// utilization of the kubeapi-server and work agent do not increase significantly.
-	//
-	// TODO expose a flag to set the worker for each controller
-	appliedManifestWorkFinalizeControllerWorkers = 10
-	manifestWorkFinalizeControllerWorkers        = 10
-	availableStatusControllerWorkers             = 10
-	manifestWorkAgentWorkers                     = 10
-)
+
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gci)